server/handlers/getSurveys.js

```diff
@@ -0,0 +1,12 @@
+const readSurveys = require('../queries/readSurveys');
+
+const getSurveys = async (req, res) => {
+  try {
+    let result = await readSurveys();
+    res.status(200).json(result);
+  } catch (err) {
+    res.status(404).json(err.message);
```

Comment:
Is 404 the appropriate error message here? You're just reading all the surveys so there should never be a 404 if it's reached the getSurveys handler. A 500 might be more appropriate?

Reply:
done